waspc/data/Generator/templates/sdk/server/_types/serialization.ts

```diff
@@ -18,7 +18,7 @@ export interface JSONObject {
 
 type PrimitiveJSONValue = string | number | boolean | undefined | null
 
-interface JSONArray extends Array<JSONValue> {}
+export interface JSONArray extends Array<JSONValue> {}
```

Comment:
After adding the `declaration: true` field to `sdk/tsconfig.json` (see https://github.com/wasp-lang/wasp/pull/1773/files#r1492341617), job definition started throwing errors such as:
```
Exported variable 'printTimeAndNumberOfTasks' has or is using name 'JSONArray' 
from external module ".../todo-typescript/.wasp/out/sdk/wasp/server/_types/serialization"
but cannot be named.
```
By reading https://github.com/microsoft/TypeScript/issues/5711 and [this StackOverflow question](https://stackoverflow.com/questions/43900035/ts4023-exported-variable-x-has-or-is-using-name-y-from-external-module-but), I tried to export the `JSONArray` interface, which solved the problem.